File: recovered/src/com/lumiyaviewer/lumiya/ui/chat/contacts/ChatterListAdapter.java
package com.lumiyaviewer.lumiya.ui.chat.contacts;

import android.content.Context;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.BaseAdapter;
import android.widget.ListAdapter;
import com.lumiyaviewer.lumiya.slproto.users.manager.UserManager;
import com.lumiyaviewer.lumiya.ui.chat.ChatterDisplayInfo;

/* loaded from: classes.dex */
abstract class ChatterListAdapter extends BaseAdapter implements ListAdapter {
    protected final Context context;
    private final LayoutInflater inflater;
    protected final UserManager userManager;
    private final ChatterItemViewBuilder viewBuilder = new ChatterItemViewBuilder();
    private boolean userDistanceInline = true;

    ChatterListAdapter(Context context, UserManager userManager) {
        this.context = context;
        this.userManager = userManager;
        this.inflater = LayoutInflater.from(context);
    }

    @Override // android.widget.Adapter
    public View getView(int i, View view, ViewGroup viewGroup) {
        Object item = getItem(i);
        if (!(item instanceof ChatterDisplayInfo)) {
            return null;
        }
        this.viewBuilder.reset();
        ((ChatterDisplayInfo) item).buildView(this.context, this.viewBuilder, this.userManager);
        return this.viewBuilder.getView(this.inflater, view, viewGroup, this.userDistanceInline);
    }

    void setUserDistanceInline(boolean z) {
        this.userDistanceInline = z;
    }
}

File: app/src/main/java/com/lumiyaviewer/lumiya/ui/chat/contacts/ChatterListAdapter.kt
package com.lumiyaviewer.lumiya.ui.chat.contacts

import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.BaseAdapter
import android.widget.ListAdapter
import com.lumiyaviewer.lumiya.slproto.users.manager.UserManager
import com.lumiyaviewer.lumiya.ui.chat.ChatterDisplayInfo

internal abstract class ChatterListAdapter(
    protected val context: Context,
    protected val userManager: UserManager
) : BaseAdapter(), ListAdapter {
    private val inflater: LayoutInflater = LayoutInflater.from(context)
    private val viewBuilder = ChatterItemViewBuilder()
    private var userDistanceInline = true

    override fun getView(i: Int, view: View?, viewGroup: ViewGroup): View? {
        val item = getItem(i)
        if (item !is ChatterDisplayInfo) {
            return null
        }
        viewBuilder.reset()
        item.buildView(context, viewBuilder, userManager)
        return viewBuilder.getView(inflater, view, viewGroup, userDistanceInline)
    }

    fun setUserDistanceInline(userDistanceInline: Boolean) {
        this.userDistanceInline = userDistanceInline
    }
}
